integration/helpers/create-fixture.tsx

```diff
@@ -399,6 +408,18 @@ export async function getHtml(page: Page, selector?: string) {
   return selector ? selectHtml(html, selector) : prettyHtml(html);
 }
 
+export function getAttribute(
+  html: string,
+  selector: string,
+  attributeName: string
+) {
+  let el = cheerio(selector, html);
+  if (!el.length) {
+    throw new Error("Invalid HTML");
```

Comment:
Seems like instead of saying "invalid HTML" we could say something like "no element matches selector X" or something.

Reply:
Updated